GitCommands/Git/GitModule.cs

```diff
@@ -2964,7 +2964,7 @@ public IReadOnlyList<string> GetMergedRemoteBranches()
             var remotes = GetRemoteNames();
 
             return _gitExecutable
-                .GetOutputLines(GitCommandHelpers.MergedBranchesCmd(includeRemote: true))
+                .GetOutputLines(GitCommandHelpers.MergedBranchesCmd(includeRemote: true), throwOnErrorOutput: false)
```

Comment:
Your broken HEAD ref is not a sufficient rationale for suppressing the error handling.